.github/workflows/CI.yml

```
@@ -43,6 +43,12 @@ jobs:
           # Add a few macOS jobs (not too many, the number we can run in parallel is limited)
           - julia-version: 'nightly'
             os: macOS-latest
+          # Add a job on the local runner
+          - julia-version: '1.9'
+            #os doesn't need to be a real OS. Its just the variable which is evaluated for runs-on
+            # right ?
+            os: rptu-1
+            group: long
```

```suggestion
            group: ''
```
I think it would make sense to set it to empty for this job. Then all tests are run instead of just one of the two subsets.
(Especially the long test-group doesn't parallelize that well as it contains only 17 files)